src/main/java/org/folio/rest/impl/PatronServicesResourceImpl.java

```diff
@@ -1167,4 +1177,42 @@ private String buildQueryWithRequesterId(String requesterId, String sortBy) {
     return format("(requesterId==%s and status==Open*)", requesterId);
   }
 
+  private CompletableFuture<Boolean> isEcsTlrFeatureEnabled(VertxOkapiHttpClient httpClient,
```

Comment:
That's a lot of code specific to settings. Can we extract it into a separate service, e.g. `SettingsService`?